bundles/de.uka.ipd.sdq.codegen.simucontroller/src/de/uka/ipd/sdq/pcm/transformations/ApplyConnectorCompletionsJob.java

```diff
@@ -133,22 +133,22 @@ private IFolder getOrCreateCompletionFolder() throws JobFailedException{
      */
     private void addMiddleware(PCMAndCompletionModelHolder models, ResourceContainer resContainer) {
         AssemblyContext ctx = CompositionFactory.eINSTANCE.createAssemblyContext();
-        ctx.setEntityName("AssCtx Middleware " + resContainer.getEntityName());
+        ctx.setEntityName("AssCtx Middleware " + resContainer.getEntityName() + " " + resContainer.getId());

```

Comment:
Do you add a blank for the AssemblyContext and not for the AllocationContext on purpose?

Reply:
No, there is no reason for that. I added the missing space.